packages/gatsby-plugin-functions/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tsconfig.json"
```

Comment:
this should rely on the user's typescript config in the default webpack setup

Reply:
True, good call!

Reply:
this is necessary for our build so we need this